### PR TITLE
Section "Footer"

### PR DESCRIPTION
**Pull request by olegsl2011**

### CODE REVIEW

**Comment by olegsl2011** [comment]
fix merge conflict

File: src/partials/footer.html [hunk 1]
```diff
@@ -1,5 +1,47 @@
-<footer class="footer">
-  <div class="container">
-    footer
+<footer>
+  <div class="footer-container">
```
Comment (Bignichok) [inline src/partials/footer.html]: please, use common `.container` class for your footer

File: src/partials/footer.html [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+      <div class="footer-logo">
+          <h1>english<span style="color: #4DC274;">excellence</span>.</h1>
```
Comment (Bignichok) [inline src/partials/footer.html]: do not use inline styles, add styling via css classes

File: src/partials/footer.html [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+          <div class="contact-info">
+              <p><span class="label">Phone number:</span></p>
+              <p><a href="tel:+380684439426" class="phone-number">+380 (68) 443-94-26</a></p>
+              <p><span class="label">Address:</span></p>
+              <p><a href="https://www.google.com/maps?q=32a+M.+Bazhana+ave" target="_blank" class="address">32a M. Bazhana ave</a></p>
+          </div>
```
Comment (Bignichok) [inline src/partials/footer.html]: you should use address tag here

File: src/css/footer.css [hunk 4]
```diff
@@ -1,3 +1,295 @@
...
+  padding: 20px 40px;
+  background-color: #1a1a1a;
+  color: white;
+  align-items: flex-start;
 }
+.footer-all {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 40px;
+  background-color: #1a1a1a;
```
Comment (Bignichok) [inline src/css/footer.css]: all colors use from css variables (base.css file)

File: src/css/footer.css [hunk 5]
```diff
@@ -1,3 +1,295 @@
...
+  color: white;
+  align-items: flex-start;
+}
+
+.contact-info .label {
+  font-size: 1em;
```
Comment (Bignichok) [inline src/css/footer.css]: use all fon sizes in px from figma

File: src/css/footer.css [hunk 6]
```diff
@@ -1,3 +1,295 @@
...
+  font-weight: 400;
+  color: #888888;
+  margin-bottom: 5px;
```
Comment (Bignichok) [inline src/css/footer.css:32]: most of your margins and paddings are incorrect, use dev mode in figma to get correct sizes

File: src/partials/footer.html (hunk 2 above)
Comment (Bignichok) [inline src/partials/footer.html]: it cant be 'h1', because we have only one main titile on page

**Review by Bignichok — APPROVED** [review]

**Pull request merged**